src/commands/bootstrap/index.ts

```diff
@@ -485,12 +485,27 @@ const bootstrap = async (props: CommonProps) => {
 
   if (finalOptions.framework === 'Next.js') {
     let template;
-    if (finalOptions.auth === 'auth.js') {
+    if (finalOptions.auth === 'auth.js' && finalOptions.orm === 'drizzle') {
       template =
         'https://github.com/neondatabase/neonctl-create-app-templates/tree/main/next-drizzle-authjs';
-    } else {
+    } else if (
+      finalOptions.auth === undefined &&
+      finalOptions.orm === 'drizzle'
+    ) {
       template =
         'https://github.com/neondatabase/neonctl-create-app-templates/tree/main/next-drizzle';
+    } else if (
+      finalOptions.auth === 'auth.js' &&
+      finalOptions.orm === 'prisma'
+    ) {
+      template =
+        'https://github.com/neondatabase/neonctl-create-app-templates/tree/add-prisma-templates/next-prisma-authjs';
```

Comment:
Reminder: I need to change these to `tree/main` once https://github.com/neondatabase/neonctl-create-app-templates/pull/7 lands.